src/Tribe/Admin/Helpers.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+// Don't load directly
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+}
+
+/**
+ * Class with a few helpers for the Administration Pages
+ */
+class Tribe__Events__Admin__Helpers {
+	/**
+	 * Static Singleton Holder
+	 * @var Tribe__Events__Admin__Helpers|null
+	 */
+	protected static $instance;
+
+	/**
+	 * Static Singleton Factory Method
+	 *
+	 * @return Tribe__Events__Admin__Helpers
+	 */
+	public static function instance() {
+		if ( ! isset( self::$instance ) ) {
+			$className      = __CLASS__;
+			self::$instance = new $className;
+		}
+
+		return self::$instance;
+	}
+
+
+	/**
+	 * Matcher for Admin Pages related to Post Types
+	 *
+	 * @param  boolean|string|array|null $id What will be checked to see if we return true or false
+	 *
+	 * @return boolean
+	 */
+	public function is_post_type_screen( $post_type = true ){
```

Comment:
I'm not sure if `$post_type = true` (or any boolean) makes sense here.  I feel like `null` would be clearer.